lib/Functional/Functional.ark

```diff
@@ -1,17 +1,18 @@
 {
+
+    (import "../Math/Arithmetic.ark")
```

Comment:
You can only import `Math/Arithmetic.ark`, the compiler knows when a file belongs to the lib and when it doesn't

Reply:
Oh hopefully. I thought imports were relative (which would be painful).